spec/system/candidate_interface/references/candidate_submits_a_duplicate_application_feature_flag_disabled_spec.rb

```diff
@@ -66,6 +66,10 @@ def then_i_can_see_my_application_has_been_successfully_submitted
     choose 'Prefer not to say'
     click_button t('continue')
 
+    # Did you ever get free school meals in the UK?
+    choose 'Prefer not to say'
```

Comment:
Minor suggestion:

```suggestion
    within_fieldset 'Did you ever get free school meals in the UK?' do
      choose 'Prefer not to say'
    end
```

Reply:
I think this is sufficient? There is only likely to be one prefer not to say option on the page and it keeps with the current pattern